src/pages/journal/journal.ts

```diff
@@ -79,7 +79,7 @@ export class JournalPage extends BasePageComponent implements OnInit, OnDestroy
       testSlots$,
       // Run any transformations necessary here
       error$.pipe(map(this.showError)),
-      isLoading$.pipe(map(this.handleLoadingUI))
+      isLoading$.pipe(map(this.handleLoadingUI)),
```

Comment:
No need to use this comma

Reply:
This is just a style thing on any block/parameter list that is one-per-line. Was thinking this is something we should enforce in the linter. Probably one for the wider team.